lib/matplotlib/text.py

```diff
@@ -976,6 +976,11 @@ def set_fontfamily(self, fontname):
         strings in decreasing priority.  Each string may be either a real font
         name or a generic font class name.  If the latter, the specific font
         names will be looked up in the corresponding rcParams.
+        Defaults to `matplotlib.rcParams['font.family']`.
```

Comment:
Just leave out this line. `fontname` has no default; `None` has to be passed explicitly.

Also, please make the following text a separate paragraph by inserting an empty line here.

Reply:
Done